react-carousel/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@brainhubeu/react-carousel",
+  "version": "2.0.0-beta.1",
+  "description": "Carousel component for React",
+  "engines": {
+    "npm": ">=6.14.3"
+  },
+  "files": [
+    "lib/*"
+  ],
+  "scripts": {
+    "lint": "esw webpack.config.* src tools test --color",
+    "clean-dist": "rimraf ./lib && mkdir lib",
+    "prebuild": "npm run clean-dist",
+    "build": "node tools/build.js",
+    "test": "jest",
+    "test:coverage": "yarn jest --coverage"
+  },
+  "author": "Brainhub",
+  "license": "MIT",
+  "devDependencies": {
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.8.7",
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-transform-runtime": "^7.10.4",
+    "@babel/polyfill": "^7.8.7",
+    "@babel/preset-env": "^7.8.7",
+    "@babel/preset-react": "^7.8.3",
+    "@babel/register": "^7.8.6",
+    "autoprefixer": "^9.7.4",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.0.6",
+    "babel-plugin-transform-react-constant-elements": "^6.23.0",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
+    "chalk": "^3.0.0",
+    "classnames": "^2.2.6",
+    "css-loader": "^3.4.2",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
+    "file-loader": "^5.1.0",
+    "html-webpack-plugin": "^3.2.0",
+    "husky": "^4.2.3",
+    "jest": "^26.1.0",
+    "jest-enzyme": "^7.1.2",
+    "jsdom": "^16.2.1",
+    "lodash": "^4.17.15",
+    "node-sass": "^4.13.1",
+    "postcss-loader": "^3.0.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "resize-observer-polyfill": "^1.5.1",
+    "react-resize-detector": "^5.0.6",
+    "react-test-renderer": "^16.13.1",
+    "recoil": "^0.0.10",
+    "rimraf": "3.0.2",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.1.3",
+    "terser-webpack-plugin": "^3.0.6",
+    "url-loader": "^3.0.0",
+    "webpack": "^4.42.0"
+  },
+  "peerDependencies": {
+    "react": ">0.14.0 || >15.0.0 || >16.0.0",
```

Comment:
isn't it possible to simplify this condition?

Reply:
Since this carousel runs on hooks and requires using hooks for writing custom plugins I will change that to 
  "peerDependencies": {
    "react": "^16.8.0",
    "react-dom": "^16.8.0"
  },